op/model.py

```diff
@@ -336,17 +350,23 @@ def relation_list(self, relation_id):
                 raise RelationNotFoundError() from e
             raise
 
-    def relation_get(self, relation_id, member_name):
+    def relation_get(self, relation_id, member_name, is_app):
+        if not isinstance(is_app, bool):
+            raise RuntimeError('is_app parameter to relation_get must be a boolean')
+
         try:
-            return self._run('relation-get', '-r', str(relation_id), '-', member_name, return_output=True, use_json=True)
+            return self._run('relation-get', '-r', str(relation_id), '-', member_name, f'--app={is_app}', return_output=True, use_json=True)
```

Comment:
Hmm... is it --app, or --application?  I recall seeing it both ways. In the spec I recall suggesting --app.

Reply:
It's `--app`.

https://github.com/juju/juju/blob/juju-2.7.0/worker/uniter/runner/jujuc/relation-get.go#L78-L81